src/main/asciidoc/docs/testng_listeners.adoc

```diff
@@ -214,3 +214,146 @@ Finishing
 
 This mechanism allows you to apply the same set of listeners to an entire organization just by adding a jar file to the classpath, instead of asking every single developer to remember to specify these listeners in their `testng.xml` file.
 
+==== Ordering listeners in TestNG
+
+TestNG now allows you to control the order in which listeners are executed.
+
+This is particularly useful when you have multiple listeners that are altering the test result states and so you would like to ensure that they do so in some deterministic order.
+
+Following is how we can get this done.
+
+* To start with, you would need to build an implementation of the interface `org.testng.ListenerComparator`
+* Now this implementation needs to be plugged into TestNG via the configuration parameter `-listenercomparator`
+
+CAUTION: TestNG orders only user's listeners and which are not part of the exclusion list that can be specified via the JVM argument `-Dtestng.preferential.listeners.package` (Multiple fully qualified class names can be specified as comma separated values). If nothing is specified, TestNG by default excludes all the IntelliJ IDEA listeners under the package `com.intellij.rt.*`.
+
```

Comment:
Edit: it should be ok

Reply:
@juherr I did. When i generated the documentation, it looks like below 

<img width="969" alt="image" src="https://github.com/testng-team/testng-team.github.io/assets/909773/5df5ca96-6c3d-4860-b035-c5f5c88f6a4c">

So I ignored this comment since I wasn't sure what additional change was required in this case.